docs/pages/develop/development-builds/introduction.mdx

```diff
@@ -6,19 +6,28 @@ searchRank: 98
 searchPosition: 1
 ---
 
+import { Collapsible } from '~/ui/components/Collapsible';
 import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
 import { Step } from '~/ui/components/Step';
 import { VideoBoxLink } from '~/ui/components/VideoBoxLink';
 
-When you created your first React Native project with `npx create-expo-app` and ran it with `npx expo start`, you most likely started off using the [Expo Go](https://expo.dev/go) app for development. Expo Go is a native app that we (as in Expo) built and submitted to the [Google Play Store](https://play.google.com/store/apps/details?id=host.exp.exponent) and [Apple App Store](https://apps.apple.com/us/app/expo-go/id982107779) so you could get coding quickly. It is a sandbox app with a number of native libraries included within (see the [dependencies list](https://github.com/expo/expo/blob/main/apps/expo-go/package.json#L23)). This means that developers may update their app's JavaScript code on their local machine and see the changes on Expo Go.
+A **development build** is a debug build of your app that is optimized for quick iterations when creating an app. It contains the the [`expo-dev-client`](/versions/latest/sdk/dev-client/) library. By using development builds, you gain full control over the native runtime, so you can [install any native libraries](#use-libraries-with-native-code-that-arent-in-expo-go), [modify any project configuration](/config-plugins/introduction/), or [write your own native code](/modules/get-started/). With Expo development builds, you are building your own native app.
```

Comment:
```suggestion
"Development build" is the term that we use for a "Debug" build of an app that includes the [`expo-dev-client`](/versions/latest/sdk/dev-client/) library. This library augments the built-in React Native development tooling with additional capabilities, such as support for inspecting network requests and a "launcher" UI that lets you switch between different development servers (such as between a server running on your machine or a teammate's machine) and deployments of your app (such as published updates with EAS Update).
```

Reply:
something like this is more what i think would be useful here. people can learn more about expo go differences below in that collapsible.